src/html/header-basic-mega.html

```diff
@@ -0,0 +1,41 @@
+
+  <a class="usa-skipnav" href="#main-content">Skip to main content</a>
+  <header class="usa-header usa-header-basic-megamenu" role="banner">
+    <div class="usa-disclaimer">
+      <div class="usa-disclaimer-inner">
+        <div class="usa-disclaimer-official">
+          <img src="{{ site.baseurl }}/assets/img/favicons/favicon-40.png" alt="U.S. flag">
+          <p>An official website of the United States government</p>
+        </div>
+      </div>
+    </div>
+    <div class="usa-nav-container">
+      <div class="usa-navbar">
```

Comment:
`nav` is all the navigation links. `navbar` is no nav links. It's the extended div the logo is in that makes up the navigation bar on mobile.

Here's some screenshots of it in mobile + desktop:

<img width="818" alt="screen shot 2016-08-12 at 10 55 45 am" src="https://cloud.githubusercontent.com/assets/5249443/17632166/cc0c1b70-607b-11e6-9f5c-1d2aaa369507.png">

<img width="1145" alt="screen shot 2016-08-12 at 10 55 55 am" src="https://cloud.githubusercontent.com/assets/5249443/17632168/cf5e21e2-607b-11e6-847b-58eeeb83f2b9.png">